src/index.js

```diff
@@ -62,6 +59,10 @@ export default class Lottie extends React.Component {
   }
 
   componentDidUpdate(prevProps) {
+    if (this.props.options.loop !== undefined) {
```

Comment:
@felippenardi I noticed this change because in one of my recent project. I have code logic like this:

```
function ExampleComponent() {
  const status = pollDataFromServer()

  if (status === 'a') {
    return (
      <Lottie
        options={{ animationData: dataA, loop: false }}
      />
    )
  } 
  return (
    <Lottie
      options={{ animationData: dataB }}
    />
  )
}
```
if I don't specify `loop: true`, when status change from 'a' to something else, the animation will stop looping. I believe it is caused by the options.loop is undefined here, so the animation won't start looping